code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -21,6 +21,11 @@
 /mob/living/simple_animal/hostile/blob/blob_act()
 	return
 
+/mob/living/simple_animal/hostile/blob/CanPass(atom/movable/mover, turf/target, height=0)
```

Comment:
Spaces in default parameters. However, if you don't use them, not need to specify them (`..()` will call correctly without them).